cartridges/int_stickyio_sfra/cartridge/templates/default/checkout/shipping/shipping.isml

```diff
@@ -1,9 +1,16 @@
-<div class="shipping-section">
+<isif condition="${pdict.sfccVersion60}">
+	<div class="card ghost shipping">
+	    <div class="card-header">
+	        <h2 class="card-header-custom">${Resource.msg('heading.checkout.shipping', 'checkout', null)}</h2>
+	    </div>
+	</div>
```

Comment:
Seems like there is an extra unneeded tab on lines 2-6.

Reply:
fixed